<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pro_dayset_dlg</class>
 <widget class="QDialog" name="pro_dayset_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>輸入Pro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pro++.qrc">
    <normaloff>:/images/pro++.ico</normaloff>:/images/pro++.ico</iconset>
  </property>
  <widget class="QTextEdit" name="note_edit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="inpt_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>錄  入</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ctlpro_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="alpro_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="rpro_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="wpro_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>寫作用時+:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>閲讀調研（文史哲）用時+:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>藝術學習（電影繪畫）用時+:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>計算機科學與技術研習用時：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>備注:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pro++.qrc"/>
 </resources>
 <connections/>
</ui>
